RELEASE_NOTES.md

```diff
@@ -1,3 +1,112 @@
+## Cookstyle 5.2
+
+### New Chef Cops
+
+#### Chef/InvalidPlatformMetadata
+
+The `InvalidPlatformMetadata` cop detects invalid platform names being used in the supports metadata.rb method. It uses a hardcoded set of common typos so it's not able to detect all invalid platforms, but it does detect the most common mistakes found on the Supermarket.
+
+`Enabled by default`: True
+
+`Autocorrects`: True (deletes the occurrence)
+
+#### Chef/RespondToResourceName
+
+The `RespondToResourceName` cop detects the usage of respond_to?(:resource_name) in resources to provide backwards compatibility. This `respond_to?` check is no longer necessary in Chef Infra Client 12.5+
+
+`Enabled by default`: True
+
+`Autocorrects`: True (deletes the `if respond_to?` occurence)
+
+#### Chef/RespondToProvides
+
+The `RespondToProvides` cop detects the usage of respond_to?(:provides) in providers to provide backwards compatibility. This `respond_to?` check is no longer necessary in Chef Infra Client 12+
+
+`Enabled by default`: True
+
+`Autocorrects`: True (deletes the `if respond_to?` occurence)
+
+#### Chef/RespondToInMetadata
+
+The `RespondToInMetadata` cop detects the usage of `if respond_to?(:foo)` to gate newer metadata methods from running on older Chef Infra Client releases. In Chef Infra Client 12.15 and later this is no longer necessary as chef-client will no longer fail if unknown metadata is encoutered.
```

Comment:
```suggestion
The `RespondToInMetadata` cop detects the usage of `if respond_to?(:foo)` to prevent newer metadata methods from running on Chef Infra Client versions older than 12.15. The chef-client doesn't fail when it encounters unknown metadata in Chef Infra Client versions 12.15.
```